lib/tasks/repair_nfn.rake

```diff
@@ -11,7 +11,10 @@ namespace :data_repair do
     affected_items = ActiveRecord::Base.connection.execute(query)
 
     affected_items.each do |params|
-      user_id,  reducer_key, subgroup = params
+      user_id, reducer_key, subgroup = params.values
+
+      puts "Combining reductions for user #{user_id}"
+      puts
```

Comment:
this on purpose?

Reply:
it was (we talked about it) but in the future when using puts this way i'll use an empty string for clarity